src/assets/assets.service.ts

```diff
@@ -175,4 +175,13 @@ export class AssetsService {
       },
     });
   }
+
+  async lastUpdate(): Promise<Date | null> {
```

Comment:
It'd be great to include a small unit test for this method in `assets.service.spec.ts`

Reply:
Good point, will do!